index.js

```diff
@@ -71,6 +184,46 @@ function compareValues(parent, theirs, mine, path) {
       theirs: theirs
     };
   }
+  else if(_.isArray(parent) || _.isArray(mine) || _.isArray(theirs)) {
+    // Maintain array order unless flagged as ignoreOrder
+    if (!_.isUndefined(options) && options['ignoreOrder']) {
+      parent = parent.sort();
+      mine = mine.sort();
+      theirs = theirs.sort();
+    }
+    if (!_.isEqual(parent, theirs) && !_.isEqual(parent, mine) && !_.isEqual(theirs, mine)) {
+      // All 3 are different: 'C'
+      return {
+        kind: 'C',
+        path: path.slice(),
+        parent: parent,
+        theirs: theirs,
+        mine: mine
+      };
+    }
+    if (_.isEqual(parent, theirs) && !_.isEqual(theirs, mine)) {
+      // parent/theirs are equal, mine is different: 'E'
+      return {
+        kind: 'E',
+        path: path.slice(),
+        parent: parent,
+        theirs: theirs,
+        mine: mine
+      };
+    }
+    if (_.isEqual(parent, mine) && !_.isEqual(theirs, mine)) {
+      // parent/mine are equal, theirs is different: 'C'
+      return {
+        kind: 'C',
+        path: path.slice(),
+        parent: parent,
+        theirs: theirs,
+        mine: mine
+      };
+    } else {
+      // All 3 are equal
```

Comment:
`return;`? (Also in the other place in this function with `// All 3 are equal` (line 258 currently)